core/new-gui/src/app/workspace/service/operator-metadata/mock-operator-metadata.data.ts

```diff
@@ -65,7 +71,10 @@ export const mockOperatorSchemaList: ReadonlyArray<OperatorSchema> =
         operatorGroupName: 'View Results',
         numInputPorts: 1,
         numOutputPorts: 0,
-        userFriendlyName: 'View Results'
+        userFriendlyName: 'View Results',
+        property_description: {
+
```

Comment:
add some default data into `property_description` to check if the interface defined is working correctly.